docs/advanced-topics/message-system/clientrpc.md

```diff
@@ -17,7 +17,7 @@ darkImageSrc="/img/sequence_diagrams/RPCs/ClientRPCs_Dark.png?text=DarkMode"/>
 
 ## Declaring a ClientRpc
 
-You can declare a `ClientRpc` by marking a method with `[ClientRpc]` attribute and including the `ClientRpc` suffix in the method name.
+You can declare a ClientRpc by marking a method with `[ClientRpc]` attribute and including the `ClientRpc` suffix in the method name.
```

Comment:
> method with **the** `[ClientRpc]`
?

Reply:
This wasn't new content. Does this not make sense? Maybe we should remove it?